web/packages/teleport/src/Discover/Database/CreateDatabase/const.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export const timeoutErrorMsg =
+  'Teleport could not detect your new database in time. Please try again.';
+
+export const dbWithoutDbServerExistsErrorMsg =
+  'already exists but there are no database servers for it';
```

Comment:
```suggestion
  'already exists but there are no Teleport agents proxying it';
```

nit: I feel like the `db_server` vs `db` thing is not well understood by users, so maybe wording it like this instead is more clear?